lnwallet/chancloser/chancloser.go

```diff
@@ -89,6 +103,9 @@ type ChanCloseCfg struct {
 	// Disconnect will disconnect from the remote peer in this close.
 	Disconnect func() error
 
+	// MaxFee, is non-zero represents the highest fee that the initiator is
+	// willing to pay to close the channel.
+	MaxFee chainfee.SatPerKWeight
```

Comment:
nit: missing new line

Reply:
My latest diff has one between this and the `Quit` channel below